examples/gp/pseudovector/gpmsa_pseudovector.C

```diff
@@ -7,22 +7,24 @@
 
 #include <cstdio>
 
+#define LINE_SIZE 512
+
 // Read in data files.  Return std deviations for each output.
 std::vector<double>
 readData(const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & simulationScenarios,
          const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & simulationParameters,
          const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & simulationOutputs,
-         const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & experimentScenarios,
+         const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & /* experimentScenarios */,
          const std::vector<QUESO::SharedPtr<QUESO::GslVector>::Type> & experimentOutputs) {
   FILE * fp_in = fopen("gp/vector/dakota_pstudy.dat", "r");
   if (!fp_in)
     fp_in = fopen("dakota_pstudy.dat", "r");
   if (!fp_in)
     queso_error_msg("Cannot find dakota_pstudy.dat");
 
-  unsigned int i, id, size = 512;
+  unsigned int i, id;
   double k_tmasl, k_tmoml, k_tnrgl, k_xkwlx, k_cd, pressure;
-  char line[size];
```

Comment:
Looks this didn't make it from C99 into C++ until C++14, but we've been skating by on compiler-specific extensions?